src/ui/Omnibox/Omnibox.ts

```diff
@@ -239,6 +240,15 @@ export class Omnibox extends Component {
     numberOfSuggestions: ComponentOptions.buildNumberOption({
       defaultValue: 5,
       min: 1
+    }),
+    /**
+     * Specifies the number of characters that should be written before displaying the suggestions
+     *
+     * Default value is `0`. Minimum value is `0`.
+     */
+    characterThresholdForSuggestions: ComponentOptions.buildNumberOption({
```

Comment:
I suggest renaming this option to `querySuggestCharacterThreshold`

(just to establish a stronger relationship between this option and `enableQuerySuggestAddon`)